docs/ibc/integration.md

```diff
@@ -28,6 +28,16 @@ The first step is to add the following modules to the `BasicManager`: `x/capabil
 and `x/ibc-transfer`. After that, we need to grant `Minter` and `Burner` permissions to
 the `ibc-transfer` `ModuleAccount` to mint and burn relayed tokens.
 
+### Integrating Light Clients
+
+In order to integrate an external light client, you must take the following steps:
+
+1. Create an implementation of `module.AppModuleBasic` for the light client. Register any required interfaces
+   in the `RegisterInterfaces` function, and ensure `Name` returns the module name. All other functions can be no-ops.
+   Alternatively, you can use an existing light client implementation such as `ibctm.AppModuleBasic{}`.
+2. Register the module with `module.BasicManager`.
```

Comment:
It looks like @damiannolan will cover your second point in [this PR](https://github.com/cosmos/ibc-go/pull/2922). Because of this I can remove the custom light client bit from this PR, and use the tm client example.